Makefile

```diff
@@ -0,0 +1,33 @@
+include .env
+
+build:
+	sudo podman build -t fpdc-web $(shell pwd)
```

Comment:
Makefiles have the `$(CURDIR)` variable available to get the Makefiles's directory.

Reply:
Ah nice :+1: